data_steward/tools/import_rdr_omop.sh

```diff
@@ -85,7 +85,7 @@ for file in $cdm_files; do
   table_name=${filename%.*}
   echo "Importing ${RDR_DATASET}.${table_name}..."
   CLUSTERING_ARGS=""
-  if grep -q person_id resource_files/fields/"${table_name}".json; then
+  if grep -q person_id resource_files/schemas/"${table_name}".json; then
```

Comment:
This will yield a file not found error. Is it ok to use `schemas/cdm/clinical/` or are there other files we need to load?